opnsynid_receivable_aging_aeroo_report/wizard/wizard_receivable_aging.py

```diff
@@ -46,7 +38,7 @@ def _default_journal(self):
         string='Company',
         comodel_name='res.company',
         required=True,
-        default=_default_company_id,
+        default=lambda self: self.env.user.company_id.id,
```

Comment:
Oke gw salah ternyata.

On Fri, Apr 22, 2016 at 3:34 PM, mikevhe18 <notifications@github.com> wrote:

> In
> opnsynid_receivable_aging_aeroo_report/wizard/wizard_receivable_aging.py
> <https://github.com/open-synergy/opnsynid-accounting-report/pull/55#discussion_r60706778>
> :
>
> > @@ -46,7 +38,7 @@ def _default_journal(self):
> >          string='Company',
> >          comodel_name='res.company',
> >          required=True,
> > -        default=_default_company_id,
> > +        default=lambda self: self.env.user.company_id.id,
>
> setau gue sih gak bisa mas, gue udah pernah coba dulu. Soalnya di dalem
> fields, kalau gak pake lambda, "self"nya undefined.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/open-synergy/opnsynid-accounting-report/pull/55/files/f0d5ab0d93345a64fca652cb8f27f27a42bd51a2..d37cbeee5a6b4dd45b5ddcd57a849475875cf23a#r60706778>
>



-- 


[image: --]

Andhitia Rama
[image: https://]about.me/andhitia.r
<https://about.me/andhitia.r?promo=email_sig>